api/cds.proto

```diff
@@ -91,26 +90,27 @@ message Cluster {
   EdsClusterConfig eds_cluster_config = 3;
 
   // The timeout for new network connections to hosts in the cluster.
-  google.protobuf.Duration connect_timeout = 4;
+  google.protobuf.Duration connect_timeout = 4 [(validate.rules).duration.gt.nanos = 0];
```

Comment:
I would use an empty object here, {}, rather than referencing nanos, since I think this ignores the seconds field in Duration, see https://github.com/google/protobuf/blob/master/src/google/protobuf/duration.proto#L108.